quantum/split_common/transport.c

```diff
@@ -228,15 +244,22 @@ bool transport_master(matrix_row_t matrix[]) {
     encoder_update_raw((uint8_t *)serial_s2m_buffer.encoder_state);
 #    endif
 
+    sync_timer_update(timer_read32());
+    serial_m2s_buffer.sync_timer = sync_timer_read32();
     return true;
 }
 
 void transport_slave(matrix_row_t matrix[]) {
```

Comment:
Looks like this got messed up somehow. 

```suggestion
void transport_slave(matrix_row_t master_matrix[], matrix_row_t slave_matrix[] {
```

Reply:
good catch